### PR TITLE
Add missing data classes from recent breaches

### LINKED ISSUE #1862: Audit missing data classes w/ recent breaches so we can get them l10n-ized
Per my Slack message:

> Mostly just a reminder for myself, but we should try and audit these breach categories since it looks like a few of them are hyphenated-slugs-that-probably-need-to-be-in-ftl-files. I’ll see if I have a script for that. And if not, I’ll try and write a script for that.
> Image from iOS (3)

Turns out that I already wrote a script for this:

```sh
npx pdehaan/blurts-server-data-classes

Missing "living-costs" data class from Kreditplus breach
Missing "mothers-maiden-names" data class from Kreditplus breach
Missing "places-of-birth" data class from Kreditplus breach
Missing "spouses-names" data class from Kreditplus breach
Missing "password-strengths" data class from Scentbird breach
```

I'll try and throw a PR today, unless somebody beats me to it.

### PR DESCRIPTION
**Pull request by pdehaan**
Via <kbd>npx pdehaan/blurts-server-data-classes</kbd>:

```sh
# npx pdehaan/blurts-server-data-classes

Missing "living-costs" data class from Kreditplus breach
Missing "mothers-maiden-names" data class from Kreditplus breach
Missing "places-of-birth" data class from Kreditplus breach
Missing "spouses-names" data class from Kreditplus breach
Missing "password-strengths" data class from Scentbird breach
```

Strings copied from https://haveibeenpwned.com/api/v3/breach/Kreditplus and https://haveibeenpwned.com/api/v3/breach/Scentbird

Fixes #1862

### CODE REVIEW

File: locales/en/data-classes.ftl [hunk 1]
```diff
@@ -78,6 +80,7 @@ parenting-plans = Parenting plans
 partial-credit-card-data = Partial credit card data
 passport-numbers = Passport numbers
 password-hints = Password hints
+password-strengths = Password strengths
```
Comment (flodolo) [inline locales/en/data-classes.ftl:83]: This looks like a weird category of data class 🤷🏼

**Comment by pdehaan** [comment]
I'm a bit undecided on adding the data class linter to CI scripts. It'd cause CI failures on unrelated PRs, but also enable us to catch these sooner. So we'd have to be OK with CI possibly failing when we merge PRs until we can get the missing l10n strings in. Thoughts @lesleyjanenorton @groovecoder?

File: locales/en/data-classes.ftl (hunk 1 above)
Comment (pdehaan) [reply locales/en/data-classes.ftl:83]: Agreed! I have no idea why a company would store that in a database.

**Comment by pdehaan** [comment]
Self merging (with r+ from flodolo) to get this into the l10n pipeline.

**Pull request merged**